src/components/Form/Form.stories.tsx

```diff
@@ -446,10 +450,18 @@ export const Playground = (): ReactElement => {
 					title={text("Title", "Form Title")}
 					description={text("Description", "This is a description example")}
 					state={state}
-					fields={fields}
+					fields={
+						showSections && !showMultipleSections ? oneSectionFields : fields
+					}
 					dispatch={dispatch}
 					getFormValues={loadReady && onLoad}
-					sections={showSections && sections}
+					sections={

```

Comment:
If we change the showSections knob to be a select then we can do something like showSections === 0 ? undefined : sections.slice(0, showSections)